src/ItemTreeTraversal/CteItemTreeTraversal.php

```diff
@@ -84,4 +106,49 @@ protected function getWithExpression(): string
     {
         return 'WITH RECURSIVE';
     }
+
+    private function getRowsCommand(
+        string $name,
+        QueryInterface $baseOuterQuery,
+        bool $areParents = true,
+    ): CommandInterface {
+        if ($areParents) {
+            $cteSelectRelationName = 'parent';
+            $cteConditionRelationName = 'child';
+            $cteName = 'parent_of';
+            $cteParameterName = 'child_name';
+        } else {
+            $cteSelectRelationName = 'child';
+            $cteConditionRelationName = 'parent';
+            $cteName = 'child_of';
+            $cteParameterName = 'parent_name';
+        }
+
+        $cteSelectItemQuery = (new Query($this->database))
+            ->select('name')
+            ->from($this->tableName)
+            ->where(['name' => $name]);
+        $quoter = $this->database->getQuoter();
+        $cteSelectRelationQuery = (new Query($this->database))
+            ->select($cteSelectRelationName)
+            ->from([
+                new Expression("{$quoter->quoteTableName($this->childrenTableName)} item_child_recursive"),
```

Comment:
Why not quote table name with alias `item_child_recursive`?

Reply:
As far as I remember, this won't work in Oracle.

Reply:
If this is a bug of `db-oracle` package, then need an issue to fix this in `db-oracle`.